src/client/components/features/OffreDeStage/Déposer/Étape2Stage/StageDeposerOffreFormulaireÉtape2Stage.tsx

```diff
@@ -29,7 +31,9 @@ const UNITE = '€';
 const LONGUEUR_MAX_TITRE = 200;
 
 enum Stage {
-	DATE_DE_DEBUT = 'dateDebut',
+	DATE_DE_DEBUT_MIN = 'dateDeDebutMin',
+	DATE_DE_DEBUT_MAX = 'dateDeDebutMax',
+	DATE_DE_DEBUT_PRECISE = 'dateDeDebutPrecise',
```

Comment:
```suggestion
	IS_DATE_DE_DEBUT_PRECISE = 'isDateDeDebutPrecise',
```

Car on pourrait s'attendre à une date en voyant date de debut precise